webapp/channels/src/components/admin_console/schema_text.tsx

```diff
@@ -56,8 +52,12 @@ export default class SchemaText extends React.PureComponent {
     };
 
     renderUntranslated = () => {
-        if (this.props.isMarkdown) {
-            const html = marked(this.props.text, {
+        const {isMarkdown, text} = this.props;
+        if (isMarkdown) {
+            if (typeof text === 'object') {
+                return text;
+            }
```

Comment:
Is it correct to return the text when it is an object?

Reply:
I copied implementation from [here](https://github.com/mattermost/mattermost/pull/23614/files#diff-24583c95ed97c3e6ecf061bd0d5b581254c3087f38eb01d587100e17216fd50dR31)

Reply:
this should be good. Because i just quickly looked and couldn't find any usage of this text be an object.